eb

```diff
@@ -100,4 +101,25 @@ if [ ! -d /cvmfs/local ]; then
 fi
 export PYTHONPATH=$EASYBUILD_ROOT:$EASYBUILD_ROOT/site-packages
 export RSNT_EASYBUILD_MAGIC_COOKIE=263ca73bb634185aab1d1b41627fdbba
+
+# argument validation
+for argument in "$@"; do
+	echo $argument
+	if [[ $CC_CLUSTER == "build-node" ]]; then
+		if [[ $argument =~ .*.eb$ ]]; then
+			if [[ $argument =~ .*/.* ]]; then
+				echo "Please do not use a path to specify the easyconfig."
```

Comment:
Can you explain why this can't be used as normal user either? I don't see the problem with that, would just move this block to the below `if [[ "$CURRENT_USER" == "ebuser" ]];` chunk

Reply:
I was thinking we might as well prevent it for all staff to get used not to use absolute paths ?